ftw/solr/browser/configure.zcml

```diff
@@ -1,8 +1,15 @@
 <configure
     xmlns="http://namespaces.zope.org/zope"
+    xmlns:zcml="http://namespaces.zope.org/zcml"
     xmlns:browser="http://namespaces.zope.org/browser"
+    xmlns:theme="http://namespaces.zope.org/plonetheme.onegov"
```

Comment:
No, I was thinking `add_scss` was maybe a directive that came from `ftw.theming` or `plone.app.theming`, and that's why the plonetheme onegov namespace irritated me. But the directives are [actually definied in the`http://namespaces.zope.org/plonetheme.onegov` namespace](https://github.com/OneGov/plonetheme.onegov/blob/master/plonetheme/onegov/meta.zcml#L6-L7) in `plonetheme.onvegov` - so it's all good.

(In the end you could call the namespace *prefix* `hans`, as long as you define it as `xmlns:hans="http://correct_namespace"` and use it as `<hans:add_scss />`, all that matters is the NS)

But since that directive is only used inside the conditional block, the NS declaration should IMHO also be moved inside:

```xml
<configure zcml:condition="installed plonetheme.onegov">
    <configure xmlns:theme="http://namespaces.zope.org/plonetheme.onegov">
	    <include package="plonetheme.onegov" />
	    <theme:add_scss path="resources/search.scss" />
    </configure>
</configure>
```
(Please test again to make sure it works for both cases)